src/createViewer.js

```diff
@@ -195,13 +198,21 @@ const createViewer = (
         // Slices share the same lookup table as the volume rendering.
         // Todo use all lookup tables on slice
         if (!!store.imageUI.image) {
-          const lut = store.imageUI.lookupTableProxies[
-            store.imageUI.selectedComponentIndex
-          ].getLookupTable()
-          const sliceActors = store.imageUI.representationProxy.getActors()
-          sliceActors.forEach(actor => {
-            actor.getProperty().setRGBTransferFunction(lut)
-          })
+          const sliceActors = store.imageUI.representationProxy.getActors();
+          const dataArray = store.imageUI.image.getPointData().getScalars();
+          sliceActors.forEach((actor) => {
+            const actorProp = actor.getProperty();
+            actorProp.setIndependentComponents(1);
```

Comment:
Here and vtk.js -- should this be a boolean?

Reply:
Yes, I think that makes more sense.  Will do.

Reply:
Done in latest push to `vtk.js` PR...